src/ui_ng/src/app/config/auth/config-auth.component.html

```diff
@@ -19,50 +19,44 @@
     <section class="form-block" *ngIf="showUAA">
         <div class="form-group">
             <label for="uaa-endpoint" class="required">{{'CONFIG.UAA.ENDPOINT' | translate}}</label>
-            <input type="text" id="uaa-endpoint" name="uaa-endpoint" size="35"
-            [(ngModel)]="currentConfig.uaa_endpoint.value" [disabled]="!currentConfig.uaa_verify_cert.editable">
+            <input type="text" id="uaa-endpoint" name="uaa-endpoint" size="35" [(ngModel)]="currentConfig.uaa_endpoint.value" [disabled]="!currentConfig.uaa_verify_cert.editable">
         </div>
         <div class="form-group">
             <label for="uaa-id" class="required">{{'CONFIG.UAA.CLIENT_ID' | translate}}</label>
-            <input type="text" id="uaa-cid" name="uaa-client-id" size="35"
-            [(ngModel)]="currentConfig.uaa_client_id.value" [disabled]="!currentConfig.uaa_verify_cert.editable">
+            <input type="text" id="uaa-cid" name="uaa-client-id" size="35" [(ngModel)]="currentConfig.uaa_client_id.value" [disabled]="!currentConfig.uaa_verify_cert.editable">
```

Comment:
Make sure no null/undefined properties existing. The properties should have default values

Reply:
confirmd that every properties on config.ts are initiated with a default value.